File: example-app/src/main/java/ai/promoted/java/ExampleApplication.java
package ai.promoted.java;

import android.app.Application;

import ai.promoted.PromotedAi;
import ai.promoted.metrics.usecases.anomaly.ModalAnomalyActivity;

public class ExampleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        PromotedAi
                .buildConfiguration()
                .withMetricsLoggingApiKey("")
                .withMetricsLoggingUrl("")
                .initialize(this);
    }
}

File: example-app/src/main/java/ai/promoted/ExampleApplication.kt
package ai.promoted

import android.app.Application

class ExampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        PromotedAi.initialize(this) {
            metricsLoggingUrl = "https://5tbepnh11h.execute-api.us-east-2.amazonaws.com/dev/main"
            metricsLoggingApiKey = "OLpsrVSd565IQmOAR62dO9GkXUJngNo5ZUdCMV70"
            xrayEnabled = true
            loggingAnomalyContactInfo = ClientConfig.LoggingAnomalyContactInfo(
                slack = ClientConfig.LoggingAnomalyContactInfo.Slack("#snackpass-promoted"),
                email = ClientConfig.LoggingAnomalyContactInfo.Email(
                    "help+snackpass@promoted.ai"
                )
            )
        }
    }
}